### DISCUSSION TITLE
Raspberry pi Farming

### DISCUSSION THREAD

**Discussion by d33gan** [Chia-Network/chia-blockchain#5577] [Support]
Hello Chia Network team,
i have one question.... is it good when i plot all plots on my fast windows desktop computer...
and i move all plots to my raspberry pi ubuntu deskop machine and i will farm the plots there???

is there any difference when a fast computer is farming or a slow raspberry pi.... ? so to win in the lottery to receive chia coins has nothing to do with how fast the machine is right?

Thank you so much for your  efforts and your answer!
Greatings Deegan

**Comment by BrandtH22** [comment]
@d33gan , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

To answer your question, so long as the machine has the necessary resources to farm having more resources (being a faster computer) does not effect the actual farming process and yes you can plot on one machine then transfer those to another to farm them. Just make sure you have the correct farmer key on the farming machine.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
